.gitignore

```diff
@@ -18,6 +18,7 @@
 .env.test.local
 .env.production.local
 .eslintcache
+spacemesh.json
```

Comment:
this file exists?

Reply:
not anymore. was related to the google bucket upload. removing this.